iml-services/iml-device/Cargo.toml

```diff
@@ -1,25 +1,29 @@
 [package]
-name = "iml-device"
-version = "0.3.0"
 authors = ["IML Team <iml@whamcloud.com>"]
 edition = "2018"
+name = "iml-device"
+version = "0.3.0"
 
 [dependencies]
 device-types = "0.1"
 futures = "0.3"
-im = { version = "13.0", features = ["serde"] }
-iml-manager-env = { path = "../../iml-manager-env", version = "0.3" }
-iml-postgres = { path = "../../iml-postgres", version = "0.3" }
-iml-rabbit = { path = "../../iml-rabbit", version = "0.3" }
-iml-service-queue = { path = "../iml-service-queue", version = "0.3" }
-iml-tracing = { version = "0.2", path="../../iml-tracing" }
-iml-wire-types = { path = "../../iml-wire-types", version = "0.3", features = ["postgres-interop"] }
-serde = { version = "1", features = ["derive"] }
+im = {version = "13.0", features = ["serde"]}
+iml-manager-env = {path = "../../iml-manager-env", version = "0.3"}
+iml-postgres = {path = "../../iml-postgres", version = "0.3"}
+iml-rabbit = {path = "../../iml-rabbit", version = "0.3"}
+iml-service-queue = {path = "../iml-service-queue", version = "0.3"}
+iml-tracing = {version = "0.2", path = "../../iml-tracing"}
+iml-wire-types = {path = "../../iml-wire-types", version = "0.3", features = ["postgres-interop"]}
+serde = {version = "1", features = ["derive"]}
 serde_json = "1.0"
-sqlx = { git = "https://github.com/launchbadge/sqlx", default-features = false, features = ["json", "macros", "offline", "postgres", "runtime-tokio" ] }
+sqlx = {git = "https://github.com/jgrund/sqlx", branch = "workspace-prepare", default-features = false, features = ["json", "macros", "offline", "postgres", "runtime-tokio"]}
```

Comment:
Does sqlx need some updates?

Reply:
Lot's of them. We should be able to switch off my branch once 0.4 is released hopefully.